docs/examples.rst

```diff
@@ -0,0 +1,17 @@
+Examples
```

Comment:
We moved our example docs from RTD to Github's README in #1626 because we thought that "examples are all about the code", so navigating it in Github makes more sense, because individual examples have their docs as READMEs, and because when people start from Github they'll probably miss the RTD.

Having some docs in RTD and some other docs in Github can make it easy to miss one or the other part, so while I don't think there's a clearcut answer where to put docs, we should always think of this.

(Note that having the examples docs on Github is not set in stone, so we should definitely reconsider if there are good reasons for that)

Reply:
I think more generally the question is how to make best use of RTD vs. Github.

IMO we should stick to Github when possible (e.g. for viewing code / browsing files), and only use RTD for documentation.

Reply:
as discussed offline the other day, let's instead do the following:

- move `examples/README.md` to this `examples.rst`
- make a BOLD LARGE LINK in `examples/README.md` that points users to the `examples.rst`

this way we have our doc in RST and searchable and all that, but we also make sure that people browsing the repo won't miss the examples documentation

Reply:
Great idea @andsteing! I've opened #2331 to do this after we merge this, I don't want to do it right now as the PR is already too large but we should do it soon after.